ledger/participant-state/kvutils/src/main/protobuf/com/daml/ledger/participant/state/kvutils/daml_kvutils.proto

```diff
@@ -194,6 +194,14 @@ message DamlTransactionRejectionEntry {
   DamlSubmitterInfo submitter_info = 1;
   reserved 5; // was reason.maximum_record_time_exceeded
 
+  // Whether this rejection should be considered a definite answer
+  // as defined in rank-based deduplication; i.e., whether this rejection
+  // signals that no other transaction with the same change-id and equal or lower submission_rank
+  // can be accepted or rejected.
```

Comment:
To avoid confusion, I'd suggest removing mentions of rank-based command deduplication here:
```suggestion
  // Whether this rejection should be considered a definite answer; i.e., whether this rejection
  // signals that no other transaction with the same change-id can be accepted or rejected.
```